app/views/shared/_use_ccms.html.erb

```diff
@@ -1,13 +1,17 @@
 <%= page_template(page_title: t(".title_html"), head_title: t(".head_title"), template: :basic, column_width: "full") do %>
-  <%= interruption_card(heading: t(".title_html")) do %>
+  <%= interruption_card(
+        heading: t(".title_html"),
+        actions: lambda {
+          safe_join(
+            [
+              govuk_button_link_to(t("shared.use_ccms.continue_in_ccms_html"), "https://portal.legalservices.gov.uk", inverse: true, role: "button"),
+              govuk_link_to(t("shared.use_ccms.back_to_applications"), submitted_providers_legal_aid_applications_path, inverse: true),
+            ],
+          )
+        },
+      ) do %>
```

Comment:
I presume this method action call is different because of the need to change the link.
Can we not use an action partial like the other views?

Reply:
This was considered. We could but it would mean an unnested partial called `_use_ccms_actions.html.erb`, or similar, and moving the translations. It felt more icky to me but I don't mind either way so your call?

Reply:
I would expect a single pattern, therefore could recommend having the partial.

Reply:
done